cmd/minishift/cmd/util/util.go

```diff
@@ -97,3 +98,9 @@ func IsValidProfile(profileName string) bool {
 	}
 	return false
 }
+
+// IsValidProfileName return true if profile name follow ^[a-zA-Z]+[0-9]*$
+func IsValidProfileName(profileName string) bool {
+	rr := regexp.MustCompile(`^\w+$`)
```

Comment:
Can we not att the '-' as well? Then we won't have to make so many changes to the docs either.

Reply:
Yes, please. the "-" is something very needed in profile names.